adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -412,6 +420,38 @@ trait GenomicRDD[T, U <: GenomicRDD[T, U]] {
   }
 
   /**
+   * Performs a broadcast inner join between this RDD and data that has been broadcast.
+   *
+   * In a broadcast join, the left RDD is collected to the driver,
+   * and broadcast to all the nodes in the cluster (broadcastTree). The key equality
+   * function used for this join is the reference region overlap function. Since this
+   * is an inner join, all values who do not overlap a value from the other
+   * RDD are dropped. As compared to broadcastRegionJoin, this function allows the
+   * broadcast object to be reused across multiple joins.
```

Comment:
other RDD?  Should this be
...all values in this RDD who do not overlap a value from the broadcast data are dropped.